pgstore.go

```diff
@@ -19,52 +20,47 @@ import (
 type PGStore struct {
 	Codecs  []securecookie.Codec
 	Options *sessions.Options
-	path    string
-	DbMap   *gorp.DbMap
+	Path    string
+	DbPool  *sql.DB
 }
 
-// Session type
-type Session struct {
-	Id         int64     `db:"id"`
-	Key        string    `db:"key"`
-	Data       string    `db:"data"`
-	CreatedOn  time.Time `db:"created_on"`
-	ModifiedOn time.Time `db:"modified_on"`
-	ExpiresOn  time.Time `db:"expires_on"`
+// PGSession type
+type PGSession struct {
```

Comment:
I don't think so. This is internal to the package, and isn't a part of the public interface. The fact that the Gorilla Sessions struct and this one have the same name was really confusing, and this one is specific to the Postgres data, so I renamed it. I've successfully tested this PR within our project, and it continues to work just fine. All of the unit tests are passing, except for the one that requires a Travis update.